config/constants.php

```diff
@@ -376,4 +376,9 @@
       'login' => '/assets/images/auth/Login_bg.jpg',
       'inactive' => '/assets/images/auth/Login_bg.jpg',
     ],
+    'aops' => [
```

Comment:
I would rename aops to something else, it's not really that generic, just name it area_of_patrol so it's easier to call.